docs/loading/Spark-connector-starrocks.md

```diff
@@ -660,53 +663,53 @@ The following example explains how to load data into columns of the [`ARRAY`](..
 
 1. Create a StarRocks table.
 
-	In the database `test`, create a Primary Key table `array_tbl` that includes one `INT` column and two `ARRAY` columns.
-	
-	```SQL
-	CREATE TABLE `array_tbl` (
-	  `id` INT NOT NULL,
-	  `a0` ARRAY<STRING>,
-	  `a1` ARRAY<ARRAY<INT>>
-	) ENGINE=OLAP
-	PRIMARY KEY(`id`)
-	DISTRIBUTED BY HASH(`id`)
-	;
-	```
+   In the database `test`, create a Primary Key table `array_tbl` that includes one `INT` column and two `ARRAY` columns.
+
+   ```SQL
+   CREATE TABLE `array_tbl` (
+       `id` INT NOT NULL,
+       `a0` ARRAY<STRING>,
+       `a1` ARRAY<ARRAY<INT>>
+    )
+    ENGINE=OLAP
+    PRIMARY KEY(`id`)
+    DISTRIBUTED BY HASH(`id`)
+    ;
+    ```
 
 2. Write data to StarRocks.
-   
-	Because some versions of StarRocks does not provide the metadata of `ARRAY` column, the connector can not infer
-	the corresponding Spark data type for this column. However, you can explicitly specify the corresponding Spark data type of the column in the option `starrocks.column.types`. In this exapmle, you can configure the option as `a0 ARRAY<STRING>,a1 ARRAY<ARRAY<INT>>`.
-	
-	Run the following codes in `spark-shell`:
-	
-	```scala
-	val data = Seq(
-	   |  (1, Seq("hello", "starrocks"), Seq(Seq(1, 2), Seq(3, 4))),
-	   |  (2, Seq("hello", "spark"), Seq(Seq(5, 6, 7), Seq(8, 9, 10)))
-	   | )
-	val df = data.toDF("id", "a0", "a1")
-	df.write
-	     .format("starrocks")
-	     .option("starrocks.fe.http.url", "127.0.0.1:8030")
-	     .option("starrocks.fe.jdbc.url", "jdbc:mysql://127.0.0.1:9030")
-	     .option("starrocks.table.identifier", "test.array_tbl")
-	     .option("starrocks.user", "root")
-	     .option("starrocks.password", "")
-	     .option("starrocks.column.types", "a0 ARRAY<STRING>,a1 ARRAY<ARRAY<INT>>")
-	     .mode("append")
-	     .save()
-	```
+
+   Because some versions of StarRocks does not provide the metadata of `ARRAY` column, the connector can not infer the corresponding Spark data type for this column. However, you can explicitly specify the corresponding Spark data type of the column in the option `starrocks.column.types`. In this exapmle, you can configure the option as `a0 ARRAY<STRING>,a1 ARRAY<ARRAY<INT>>`.
```

Comment:
```suggestion
   Because some versions of StarRocks does not provide the metadata of `ARRAY` column, the connector can not infer the corresponding Spark data type for this column. However, you can explicitly specify the corresponding Spark data type of the column in the option `starrocks.column.types`. In this example, you can configure the option as `a0 ARRAY<STRING>,a1 ARRAY<ARRAY<INT>>`.
```